torch/distributed/rpc/__init__.py

```diff
@@ -53,7 +53,7 @@ def init_model_parallel(
             self_rank (int): a globally unique id/rank of this node.
             init_method(str): backend specific init arguments.
             num_send_recv_threads(int): Number of threads for send/recv work.
-            rpc_timeout (datetime.timedelta): Timeout for RPCs. Defaults to 10 seconds.
+            rpc_timeout (datetime.timedelta): Timeout for RPCs. Defaults to 60 seconds.
```

Comment:
This was incorrect as we decided to change the default timeout and this was not updated.